docs/resources.rst

```diff
@@ -142,8 +131,5 @@ Solidity Tools
 Third-Party Solidity Parsers and Grammars
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* `solidity-parser <https://github.com/ConsenSys/solidity-parser>`_
-    Solidity parser for JavaScript
-
-* `Solidity Grammar for ANTLR 4 <https://github.com/federicobond/solidity-antlr4>`_
-    Solidity grammar for the ANTLR 4 parser generator
```

Comment:
Is the antlr4 grammar link no longer necessary?

Reply:
We have taken it in into our repo a while ago :) See https://github.com/ethereum/solidity/pull/9587#discussion_r466399898